Assets/Scripts/Localization/LocalizationDownloader.cs

```diff
@@ -255,14 +263,8 @@ private static void OnDownloadLocalizationComplete(Action onLocalizationDownload
                 Directory.Delete(dirInfo[0].FullName);
 
                 Debug.ULogChannel("LocalizationDownloader", "New localization files successfully downloaded!");
-            }
```

Comment:
Note: Removed this because catching the exception only to spit it back out makes it much harder to find the specific line that raised it. letting the exception remain is best until it is handled properly